libs/books/feature/src/lib/book-search/book-grid/book-grid.component.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { SharedTestingModule } from '@tmo/shared/testing';
+import { BookGridComponent } from './book-grid.component';
+import { BooksFeatureModule } from '../../books-feature.module';
+
+describe('BookGridComponent', () => {
+  let component: BookGridComponent;
+  let fixture: ComponentFixture<BookGridComponent>;
+  const books = [{
+    id: `${Date.now()}a`,
+    title: `Some Title ${Math.floor(Math.random() * 9999)}a`,
+    authors: ['a', 'b', 'c'],
+    description: 'n/a',
+    isAdded: false
+  },
+  {
+    id: `${Date.now()}b`,
+    title: `Some Title ${Math.floor(Math.random() * 9999)}b`,
+    authors: ['a', 'b', 'c'],
+    description: 'n/a',
+    isAdded: false
+  }
+];
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [BooksFeatureModule, SharedTestingModule]
```

Comment:
You're only testing the grid component, you probably don't need to import the entire module. You can use something like `NO_ERRORS_SCHEMA` if all you want to check is that it repeats the book grid item, then you can use a selector for the `tmo-book-grid-item` tag itself.